jobs/ci-kubernetes-e2e-gce-gci-latest-1.5-upgrade-etcd.env

```diff
@@ -0,0 +1,15 @@
+### job-env
+
+STORAGE_BACKEND=etcd2
+ETCD_VERSION=2.2.1
+ETCD_IMAGE=2.2.1
+
+E2E_OPT=--check_version_skew=false
+E2E_UPGRADE_TEST=true
+GINKGO_UPGRADE_TEST_ARGS=--ginkgo.focus=\[Feature:EtcdUpgrade\] --etcd-upgrade-storage=etcd3 --etcd-upgrade-version=3.0.14
+JENKINS_PUBLISHED_SKEW_VERSION=ci/latest
```

Comment:
https://github.com/kubernetes/test-infra/blob/master/jenkins/e2e-image/e2e-runner.sh#L49-L69

It should be the same as JENKINS_PUBLISHED_VERSION except for skew tests AFAIK?

@krousey